pylintrc

```diff
@@ -67,4 +67,4 @@ ignore-docstrings=yes
 # (useful for modules/projects where namespaces are manipulated during runtime
 # and thus existing member attributes cannot be deduced by static analysis. It
 # supports qualified module names, as well as Unix pattern matching.
-ignored-modules=numpy,numpy.*,scipy.stats,scipy.spatial
+ignored-modules=numpy,numpy.*,scipy.stats,scipy.spatial,morphio
```

Comment:
is this really needed, I recall vaguely benoit saying that we don't need that anymore, but I'm not sure

Reply:
good, removing it